HTAN.model.csv

```diff
@@ -778,63 +776,24 @@ RPPA Validation Status,Valid = RPPA and WB correlation > 0.7; Use with Caution =
+PMID,The PubMed identifier associated with the publication (applicable to published manuscripts),,,,FALSE,Publication,,,
+Data Type,"Types of data associated with the content. Fill out Other Data Type Specified, if not on the list.","10X Visium, Bulk DNA, Bulk RNA-seq, CyCIF, H&E, IMC, LC-MS-MS, LC-MS3, MIBI, MxIF, Shotgun MS (lipidomics), mIHC, scATAC-seq, scRNA-seq, t-CyCIF, Demographics Data",,,TRUE,Publication,,,list
+Other Data Type Specified,Other types of data associated with the content.,,,,FALSE,Publication,,,list
+Supporting Link,Relevant external links associated with the content (e.g external datasets used for validation). Please note: Supporting Links and Supporting Link Descriptions are provided by authors and are not verified by the NIH NCI or the HTAN DCC. This information and any linked data should only be shared by an authorized individual(s) in accordance with the terms of the HTAN data sharing agreements and policies and/or any other applicable agreement(s).,,,,FALSE,Publication,,,
 Supporting Link Description,Description of relevant external links associated with the publication (e.g An external mouse dataset used for validation). Please note: Supporting Links and Supporting Link Descriptions are provided by authors and are not verified by the NIH NCI or the HTAN DCC. This information and any linked data should only be shared by an authorized individual(s) in accordance with the terms of the HTAN data sharing agreements and policies and-or any other applicable agreement(s).,,,,FALSE,Publication,,,
-Tool,Were any software or computational tools generated for this content,"Yes, No",,,TRUE,Publication,,,

```

Comment:
Looks like we are dropping a bunch of extra metadata attributes - are we concurrent with `main`?